### DISCUSSION TITLE
Custom theme in filament

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9604] [General]
### hello filament
first, thanks for this great repo

I make filament custom theme and in this tailwind.config.js I use color for primary like
```

import preset from '../../../../vendor/filament/filament/tailwind.config.preset'
import colors from 'tailwindcss/colors'
import forms from '@tailwindcss/forms'
import typography from '@tailwindcss/typography'
export default {
    presets: [preset],
    content: [
        './app/Filament/**/*.php',
        './resources/views/filament/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
    ],
    theme: {
        extend: {
            colors: {
                danger: colors.rose,
                primary: colors.purple,
                success: colors.green,
                warning: colors.yellow,
            },
        },
    },
}

```

but this color is only effect for text color input box focus color and other. This does not effect the button background color. why . The button color is only effect for creating color on adminPandelProvider.php ?

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/themes/#changing-the-colors

**Reply by ayemin-tun** [reply]
Can I change color on tailwind.config not admin panel?

**Reply by leandrocfe** [reply]
hum I don't know..

**Reply by coolsam726** [reply]
@ayemin-tun If I may ask, what would necessitate setting colors from tailwind.config and not via `FilamentColors` class? I personally prefer the latter because it gives me flexibility since I don't need a build step and Filament has all the tools I need to create Color Palettes on the fly.
